tests/io_utils.go

```diff
@@ -117,11 +120,50 @@ func FixErrorDevice(nodeName string) {
 
 }
 
+func executeDeviceMapperOnNode(nodeName string, cmd []string) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+
+	// Image that happens to have dmsetup
+	image := fmt.Sprintf("%s/virt-launcher:%s", flags.KubeVirtRepoPrefix, flags.KubeVirtVersionTag)
```

Comment:
We have a testimage where we could add the tool (if not already there). That would make the tests on the long run less dependant on the content of `virt-launcher` and `virt-handler`.

Reply:
You can add rpms by changing the testimage install list in `hack/rpm-deps.sh` and then run `make rpm-deps`.

Reply:
It's already in the base test image. I switched to using vm-killer as an image, which is based on it, if that's ok.